src/core/server/rendering/types.ts

```diff
@@ -129,29 +115,13 @@ export interface IRenderingProvider {
    * @param injectedVarsOverrides Provide optional variables to inject in the page.
    * @param includeUserProvidedConfig Optionally disable injecting user-provided settings in the page
    */
-  render(pluginId?: string, includeUserProvidedConfig?: boolean): Promise<string>;
+  render(pluginId?: string, options?: { includeUserSettings?: boolean }): Promise<string>;
```

Comment:
Can we rename `pluginId` to `legacyAppId`? Also it looks like the TSDocs need to be updated here.